jetty-server/src/main/java/org/eclipse/jetty/server/Connector.java

```diff
@@ -102,4 +104,48 @@
      * @return The connector name or null.
      */
     String getName();
+
+    /**
+     * Get the Local Address of the connection
+     *
+     * @return Returns the connection local address.
```

Comment:
```suggestion
     * @return Returns the connection local address or null.
```
I really don't like the or null.